backend/backend/settings.py

```
@@ -1,91 +1,164 @@
+if ENVIRONMENT == 'PROD':
+    # Production Database
+    tmpPostgres = urlparse(os.getenv("DATABASE_URL"))
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.postgresql',
+            'NAME': tmpPostgres.path.replace('/', ''),
+            'USER': tmpPostgres.username,
+            'PASSWORD': tmpPostgres.password,
+            'HOST': tmpPostgres.hostname,
+            'PORT': 5432,
+        }
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)